CRM/Export/BAO/Export.php

```diff
@@ -1214,9 +1189,6 @@ public static function mergeSameHousehold($exportTempTable, &$headerRows, &$sqlC
     foreach ($replaced as $from => $to) {
       $clause[] = "$from = $to ";
       unset($sqlColumns[$to]);
-      if ($key = CRM_Utils_Array::key($to, $allKeys)) {
-        unset($headerRows[$key]);
```

Comment:
Not needed as we don't set it in https://github.com/civicrm/civicrm-core/pull/12579/files#diff-2e1de3666cf0b15e640b507b1ab5941fR1416